code.py

```diff
@@ -0,0 +1,11 @@
+import json
+
+
+def lambda_handler(event, context):
+    if "body" not in event:
+        return "no body found"
+
+    # 1. Parse GitHub Webhook
+    payload = json.loads(event["body"])
+    print(payload)
```

Comment:
> [!CAUTION]
> The function always returns 'ok' regardless of the processing result, and also returns it as a plain string instead of a proper API Gateway response.

> [!TIP]
> Return a proper API Gateway response with a 200 status code and JSON formatted body.

```python
return {
    'statusCode': 200,
    'headers': {'Content-Type': 'application/json'},
    'body': json.dumps({'status': 'ok'})
}
```